Makefile.core.mk

```diff
@@ -437,8 +437,8 @@ security-test:
 common-test: build
 	go test ${T} ./pkg/...
 	go test ${T} ./tests/common/...
+	go test ${T} ./tools/istio-iptables/...
 	# Execute bash shell unit tests scripts
-	./tests/scripts/scripts_test.sh
 	./tests/scripts/istio-iptables-test.sh
```

Comment:
in future we should make this `go` code. Even if its just calling out to shell, for consistency with tooling